Formula/mapnik.rb

```diff
@@ -1,9 +1,8 @@
 class Mapnik < Formula
   desc "Toolkit for developing mapping applications"
   homepage "http://www.mapnik.org/"
-  url "https://github.com/mapnik/mapnik/archive/v3.0.9.tar.gz"
-  sha256 "f0242606096e2c4ca2cd0caac1ff0fd5f8054a38b5f288ba38b0e397b5b311b2"
-  revision 1
+  url "https://github.com/mapnik/mapnik/releases/download/v3.0.12/mapnik-v3.0.12.tar.bz2"
```

Comment:
@ilovezfs - thanks that is exactly the situation here. I am the maintainer of Mapnik and made specific changes upstream to enable homebrew to get back on the release tarball. The archive version automatically created by github is not viable - it does not include all the things needed to built easily and we don't want people to use it.

As far as .gz vs .bz2 - I changed the .bz2 because that is all we provide upstream. I can provide .gz if that is important for some reason but I would prefer not to maintain two compression formats unless absolutely necessary.

So, to recap, the new url is necessary.